src/foam/nanos/cron/CronSchedule.js

```diff
@@ -54,14 +59,19 @@ foam.CLASS({
       name: 'dayOfWeek',
       value: -1,
       documentation: `Day of week to execute script.
-          Ranges from 0 - 6, where 0 is Sunday. -1 for wildcard`
+          Ranges from 1 - 7, where 1 is Monday (matches java time dayofweek). -1 for wildcard`
```

Comment:
Do we need to migrate any old data or adapt so that 0 is converted to 7?

Reply:
"Admin Access Configuration" running only on ablii.